### ISSUE TITLE
WireRun doesn’t import descriptor.proto at the right phase

### ISSUE THREAD

**Issue by swankjesse** [square/wire#1246]
It crashes before we add it.

```
java.lang.IllegalArgumentException: unable to resolve 1 import:
  google/protobuf/descriptor.proto
searching 9 proto paths:
  ...
    at com.squareup.wire.schema.NewSchemaLoader.load(NewSchemaLoader.kt:107)
    at com.squareup.wire.schema.WireRun.execute(WireRun.kt:148)
    at com.squareup.wire.schema.WireRun.execute$default(WireRun.kt:145)
    at com.squareup.wire.gradle.WireTask.generateWireFiles(WireTask.kt:90)
```

**Comment by aleien** [comment]
Hello!
I am having similar issue with 'google/protobuf/descriptor.proto' (at least [it looks](https://stackoverflow.com/questions/60134172/using-protobuf-imports-i-e-google-protobuf-timestamp-proto-with-wire) like it is similar)

If it is, is there any correct workaround for this issue? We do heavily use google dependencies in our proto files.

**Comment by oldergod** [comment]
It's been fixed I believe in 3.1 or 3.2.
